core/backend/src/test/standalone/RenderMaterialElement.test.ts

```diff
@@ -144,6 +144,27 @@ describe("RenderMaterialElement", () => {
       test({ patternMap }, { Map: { Pattern: patternMap } });
     });
 
+    it("pattern map with constant lod parameters", () => {
```

Comment:
Can you add a test for normal map with constant LOD?

Reply:
done